### DISCUSSION TITLE
I'm creating a user, a service account for Zabbix authentication

### DISCUSSION THREAD

**Discussion by ithakaa** [lldap/lldap#767] [General]
I can successfully authenticate the Zabbix user, but when I utilise that user as the Zabbix service account, I encounter difficulties authenticating any other users through Zabbix. 

Is it necessary for the service account to possess administrative permissions?

**Comment by nitnelave** [comment]
Hi! I don't know about Zabbix, but in general if you have a service account, it should be in one of these groups:
- lldap_strict_readonly: allow reading info for any user
- lldap_password_manager: reading + setting a user's password
- lldap_admin: full read/write permissions

Otherwise, basic users are only allowed to read/write their own info.
